e2e/plotting.spec.ts

```diff
@@ -68,7 +68,7 @@ test('plots a time vs shotnum graph and change the plot colour', async ({
   const chart = await popup.locator('#my-chart');
   const dimensions = await chart.boundingBox();
   expect(
-    await popup.screenshot({
+    await chart.screenshot({
```

Comment:
By changing this to `chart`, I've managed to more deterministicly capture the plot itself for a screenshot. Without this, some elements of the chart are sometimes missed in the screenshot, such as the axes labels